src/script-manager/common.ts

```diff
@@ -134,12 +139,20 @@ export function normalizeParams(raw: RecaptchaParams): string[][] {
 export function toStringPair(params: RecaptchaParams): string[][] {
   return Object.entries(params).filter((pair): pair is [string, string] => typeof pair[1] === 'string')
 }
-
 export function checkRecaptchaLoad() {
-  if (typeof window === 'undefined') return false
+  if (typeof window === 'undefined') return false;
+
+  const isLoaded = () => 'grecaptcha' in window && (
+    'execute' in window.grecaptcha || 
+    'execute' in (window.grecaptcha?.enterprise ?? {})
+  );
 
-  const isLoaded = Object.hasOwn(window, 'grecaptcha') && Object.hasOwn(window.grecaptcha, 'execute')
-  if (isLoaded) recaptchaLoaded.resolve()
+  const interval = setInterval(() => {
+    if (isLoaded()) {
+      recaptchaLoaded.resolve();
+      clearInterval(interval);
```

Comment:
The design here is rely on the `onload` callback from [reCAPTCHA](https://developers.google.com/recaptcha/docs/display#explicit_render). So I didn't put a polling here. Does the `onload` callback not work with enterprise version?